multibranch/src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -76,10 +80,22 @@ public WorkflowMultiBranchProject(ItemGroup parent, String name) {
         }
 
         @Override public List<SCMDescriptor<?>> getSCMDescriptors() {
-            // TODO this is a problem. We need to select only those compatible with Workflow.
-            // Yet SCMDescriptor.isApplicable requires an actual Job, whereas we can only pass WorkflowJob.class!
-            // Can we create a dummy WorkflowJob instance?
-            return SCM.all();
+            throw new IllegalStateException("TODO unused: https://github.com/jenkinsci/branch-api-plugin/pull/24");
```

Comment:
Maybe better to just return an empty list and log the error.
It's a public API, so no idea how it could be used by external plugins and groovy scripts